Documentation/Blazorise.Docs/Pages/Blog/2022-08-31_CSSTipsForDevelopers/Index.md

```diff
@@ -0,0 +1,105 @@
+﻿---
+title: Practical CSS Tips For Developers
+description: 
+permalink: blog/practical-css-tips-for-developers
```

Comment:
The links here must always start with `/`. In this case `/blog/practical-css-tips-for-developers`.